tests/suites/test_suite_ssl.function

```diff
@@ -3737,17 +3791,23 @@ exit:
 /* END_CASE */
 
 /* BEGIN_CASE depends_on:MBEDTLS_X509_CRT_PARSE_C:!MBEDTLS_USE_PSA_CRYPTO:MBEDTLS_PKCS1_V15:MBEDTLS_RSA_C:MBEDTLS_ECP_DP_SECP384R1_ENABLED */
-void handshake_version( int version, int dtls )
+void handshake_version( int dtls, int client_min_version, int client_max_version, int server_min_version, int server_max_version, int expected_version )
```

Comment:
Minor/style: could we break this up to lines of less than 80 characters?

Reply:
Doh, my editor was set on 100 chars from previous work. Done.